tests/unit/test_bpm_calibration.py

```diff
@@ -0,0 +1,101 @@
+from pathlib import Path
+import numpy as np
+import pandas as pd
+import pathlib
+import pytest
+
+import tfs
+from generic_parser.dict_parser import ArgumentError
+from pylhc import bpm_calibration as calibration
+
+INPUTS_DIR = Path(__file__).parent.parent / 'inputs' / 'calibration'
+MEASUREMENTS_BETA = INPUTS_DIR / 'measurements' / 'for_beta'
+MEASUREMENTS_DISPERSION = INPUTS_DIR / 'measurements' / 'for_dispersion'
+EXPECTED_OUTPUT = INPUTS_DIR / 'output'
+
+
+def test_calibration_same_betabeat(tmp_path):
+    factors = calibration.main(input_path=MEASUREMENTS_BETA,
+                               output_path=tmp_path,
+                               ips=[1,4,5])
+
```

Comment:
a further test using (fake) ideal data would be good, i.e. beta_amp = beta_phase in all cases, such that all calib factors should be 1 .. as a small sanity check for one IP only

Reply:
I added a test where beta_phase == beta_amp for all BPMs, the calibration factors expect to be 1.